arviz/data/io_pymc3.py

```diff
@@ -34,47 +42,41 @@ def __init__(
             )
             else None
         )
+        if self.observations is not None:
+            self.observations = {obs.name: obs.observations for obs in self.model.observed_RVs}
 
         import pymc3
 
         self.pymc3 = pymc3
 
+    def log_likelihood_vals_point(self, point, var, log_like_fun):
+        """Compute log likelihood for each observed point."""
+        log_like_val = np.atleast_1d(log_like_fun(point))
```

Comment:
Why atleast_1d? If the variable is a scalar, then so should the logp. We shouldn't just invent dimensions.

Reply:
I'll check the behaviour too.